src/Dev/Deprecation.php

```diff
@@ -234,14 +234,22 @@ public static function get_enabled()
 
     public static function isEnabled(): bool
     {
+        // Return early if disabled
```

Comment:
```suggestion
        // Return early if set and disabled
```

Reply:
This is intended to be for _any_ disabled - i.e. we're returning early if it's disabled either via the static property _or_ the env variable. The static property is always set, so adding "set and" here makes it read (at least to me) like the comment is only about the env variable.